<?xml version="1.0" encoding="UTF-8" ?>
<testsuites disabled="0" errors="0" failures="1" tests="2" time="5.404">
 <testsuite name="demo calculator tests" timestamp="2020-01-08T15:58:02" hostname="localhost" time="5.404" errors="0" tests="2" skipped="0" disabled="0" failures="1">
  <testcase classname="demo calculator tests" name="addition test" time="5.015" />
  <testcase classname="demo calculator tests" name="subtraction test" time="0.387">
   <failure type="exception" message="Failed: homepage.clickGo1 is not a function"><![CDATA[TypeError: homepage.clickGo1 is not a function
    at UserContext.<anonymous> (/Users/ppanwar5/Documents/Protractor/ProtractorDemo/tests/calculator.js:37:18)
    at /Users/ppanwar5/.nvm/versions/node/v6.11.1/lib/node_modules/protractor/node_modules/jasminewd2/index.js:112:25
    at new ManagedPromise (/Users/ppanwar5/.nvm/versions/node/v6.11.1/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:1077:7)
    at ControlFlow.promise (/Users/ppanwar5/.nvm/versions/node/v6.11.1/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2505:12)
    at schedulerExecute (/Users/ppanwar5/.nvm/versions/node/v6.11.1/lib/node_modules/protractor/node_modules/jasminewd2/index.js:95:18)
    at TaskQueue.execute_ (/Users/ppanwar5/.nvm/versions/node/v6.11.1/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)
    at TaskQueue.executeNext_ (/Users/ppanwar5/.nvm/versions/node/v6.11.1/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3067:27)
    at asyncRun (/Users/ppanwar5/.nvm/versions/node/v6.11.1/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2927:27)
    at /Users/ppanwar5/.nvm/versions/node/v6.11.1/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:668:7
    at process.internalTickCallback (internal/process/next_tick.js:77:7)
From: Task: Run it("subtraction test") in control flow
    at UserContext.<anonymous> (/Users/ppanwar5/.nvm/versions/node/v6.11.1/lib/node_modules/protractor/node_modules/jasminewd2/index.js:94:19)
    at attempt (/Users/ppanwar5/.nvm/versions/node/v6.11.1/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4297:26)
    at QueueRunner.run (/Users/ppanwar5/.nvm/versions/node/v6.11.1/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4217:20)
    at QueueRunner.execute (/Users/ppanwar5/.nvm/versions/node/v6.11.1/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4199:10)
    at Spec.queueRunnerFactory (/Users/ppanwar5/.nvm/versions/node/v6.11.1/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:909:35)
    at Spec.execute (/Users/ppanwar5/.nvm/versions/node/v6.11.1/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:526:10)
    at UserContext.fn (/Users/ppanwar5/.nvm/versions/node/v6.11.1/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:5340:37)
    at attempt (/Users/ppanwar5/.nvm/versions/node/v6.11.1/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4297:26)
    at QueueRunner.run (/Users/ppanwar5/.nvm/versions/node/v6.11.1/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4217:20)
    at runNext (/Users/ppanwar5/.nvm/versions/node/v6.11.1/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4257:20)
From asynchronous test: 
Error
    at Suite.<anonymous> (/Users/ppanwar5/Documents/Protractor/ProtractorDemo/tests/calculator.js:26:5)
    at addSpecsToSuite (/Users/ppanwar5/.nvm/versions/node/v6.11.1/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1107:25)
    at Env.describe (/Users/ppanwar5/.nvm/versions/node/v6.11.1/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1074:7)
    at describe (/Users/ppanwar5/.nvm/versions/node/v6.11.1/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4399:18)
    at Object.<anonymous> (/Users/ppanwar5/Documents/Protractor/ProtractorDemo/tests/calculator.js:3:1)
    at Module._compile (internal/modules/cjs/loader.js:722:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:733:10)
    at Module.load (internal/modules/cjs/loader.js:620:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:560:12)]]>
   </failure>
  </testcase>
 </testsuite>
</testsuites>